README.md

```diff
@@ -118,6 +118,8 @@ Parsing [linguist/samples](https://github.com/github/linguist/tree/master/sample
 
 * [Heuristics for ".rno" extension](https://github.com/github/linguist/blob/3a1bd3c3d3e741a8aaec4704f782e06f5cd2a00d/lib/linguist/heuristics.yml#L365) in RUNOFF could not be parsed, due to unsupported lookahead in RE2 regexp engine.
 
+* [Heuristics for ".ino" extension](https://github.com/github/linguist/blob/f0e2d0d7f1ce600b2a5acccaef6b149c87d8b99c/lib/linguist/heuristics.yml#L222) in NASL could not be parsed, due to unsupported possessive quantifier in RE2 regexp engine.
+
```

Comment:
Nice catch! Wonder why this does not require adding any [corner case to the tests](https://github.com/go-enry/go-enry/blob/1ab8148c10379417036bc9d403386a8291ac63dc/common_test.go#L431) 🤔

Reply:
all test cases are automatically used from linguist and does not need to be synced somehow?

Reply:
> all test cases are automatically used from linguist

that particular test mimics one the Linguist testing strategies by verifying the results of language prediction over all original Linguist files under `./samples`. Some of the incompatible regexps e.g in the heuristics matcher make some samples fail from time to time. 

I checked, and in this case https://github.com/github/linguist/pull/4792 there is no mention of `.ino` extension at all - I guess it's a typo in README and should be `.inc` instead?

I guess the answer to original question would be that the test
```
$ ENRY_TEST_REPO="${PWD}/.linguist/" go test . -run Test_EnryTestSuite -testify.m TestLinguistCorpus
```
pass, as all those 2 `.inc` examples of the NASL are not disambiguated by the heuristic strategy `GetLanguagesByContent` (that uses those regexps), but rather are ranked first by the very last BayesianClassifier instead.

Thus I would append `.inc` to the list of other similar cases in [this comment](https://github.com/go-enry/go-enry/blob/1ab8148c10379417036bc9d403386a8291ac63dc/common_test.go#L433) to simplify the maintenance once we have a failing tests :)

Reply:
yes, it is typo, should be `.inc`, will do